### PR TITLE
feat: edits to catalogue model

### PR DESCRIPTION
**Pull request by BrendaHijmans**
What are the main changes you did:
-Edits to descriptions in several tables.
- Variable values model update

- Edit Resource model
   - Add Resources.dateLastRefresh
   - Delete Resources.dateEstablished
   - Rename Resources.startDataCollection to  Resources.endYear
   
- Change order of some attributes in Resources table

how to test:
- explain here what to do to test this (or point to unit tests)

todo:
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

File: data/_models/shared/Contacts.csv [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+Contacts,,prefix,,,,,,,,,,"Surname prefix, if applicable","DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,initials,,,,,,,,,,Initials of the contact person,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,title,ontology,,,CatalogueOntologies,Titles,,,,foaf:title,Title of the contact person,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
```
Comment (mswertz) [inline data/_models/shared/Contacts.csv:12]: you want to do the 'pre' and 'post' titles later?

**Review by mswertz — APPROVED** [review]
I notice we really should create a new column type that understands 'other' and then allows custom terms to be added to the list when 'other' is selected (like a special flavor of ontology)

File: data/_models/shared/Contacts.csv (hunk 1 above)
Comment (BrendaHijmans) [reply data/_models/shared/Contacts.csv:12]: They all go under 'title' I think? 'prefix' is meant for e.g. 'van der' in 'van der Velde'

Comment (mswertz) [reply data/_models/shared/Contacts.csv:12]: ah no. You have titles before, such as 'Prof.dr.' and there are pre-nominal letters such as 'PhD'
see https://rapidbi.com/post-nominal-letters-and-qualifications/
but I don't mind to revisit this later.

Comment (BrendaHijmans) [reply data/_models/shared/Contacts.csv:12]: Yes I understood. I would like to do this later, it is not within the scope of the migrations.

Comment (hslh) [reply data/_models/shared/Contacts.csv:12]: I made a separate issue for this here: https://github.com/molgenis/GCC/issues/716

**Comment by BrendaHijmans** [comment]
> The model is updated ( fields are renamed, removed and added ) therefore the version should be bumped to 4.1 and migrations steps/scripts should be added

I reckoned this was still part of setting up data model 4.0, since it is not in production yet. The migrations to 4.0 are in a different pull request, it is not practical to have them in this pull request. Also, we will not migrate from 3.x to 4.0 and then again to 4.1

File: data/_models/shared/Contacts.csv (hunk 1 above)
Comment (BrendaHijmans) [reply data/_models/shared/Contacts.csv:12]: great :)

**Pull request merged**